common/lib/xmodule/xmodule/assets/vertical/public/js/vertical_student_view.js

```diff
@@ -24,32 +25,46 @@ window.VerticalStudentView = function(runtime, element) {
             apiUrl: $bookmarkButtonElement.data('bookmarksApiUrl')
         });
     });
-    $(element).find('.vert').each(
-        function(idx, block) {
-            var blockKey = block.dataset.id;
-
-            if (!block.dataset.completableByViewing) {
-                return;
+    RequireJS.require(['bundles/ViewedEvent'], function() {
```

Comment:
@cpennington: So I think this is what broke things in the prod environment because RequireJS isn't aware of Webpack-optimized prod bundle. Will this problem go away with your XModule Webpack conversion work?

Reply:
I'm not going to make RequireJS know about webpack bundles.

Reply:
😱  My apologies. The idea of making RequireJS understand Webpack is horrifying, and I had no intention of trying to push us in that direction. I'm asking about the other way -- if it's all going to be under Webpack, will it be straightforward to replace this with the equivalent call and have it all bundled together with the VerticalBlock's other JS via Webpack?

Reply:
@ormsbee or @cpennington Any updated thoughts about this?

Reply:
@iloveagent57: I'm trying a stupid/silly fix of having Webpack preserve the non-hashed version versions of the necessary assets in the prod build so that we can get this through without having to make deeper changes to make RequireJS understand anything about Webpack. I should hopefully have a PR by end of day.